### ISSUE TITLE
`require` doesn't work in REPL

### ISSUE THREAD

**Issue by ghost** [anko/eslisp#45]
`require` doesn't work in the REPL:

```
> (require "./whatever.js")
evalmachine.<anonymous>:1
require('./whatever.js');
^
ReferenceError: require is not defined
```

However the binding macro example (which is using `require`) in **How eslisp macros work (a tutorial)** works fine in the REPL.
(**jisp** already has `require` in the REPL, but it doesn't have quoting and other important stuff)

**Comment by anko** [comment]
Thanks for reporting.

I'd forgotten to set the `useGlobal`-option to [`repl.start`](https://nodejs.org/api/repl.html#repl_repl_start_options), so all code entered into the REPL ran with a `global` object that didn't have the Node-specific globals (e.g. `require`) bound.

This should be fixed in 0.7.4.

**Comment by ghost** [comment]
Thanks a lot for fixing so quickly. Really nice finally trying a JS lisp that has proper quasiquoting!
